_includes/checklist.md

```diff
@@ -1,41 +1,91 @@
         * [ ] Resolve any visible security issues, re-running the scan as needed
         * [ ] Add the issue-free scan report or [documentation about false positives](https://pages.18f.gov/before-you-ship/security/dynamic-scanning/#caveats) to the ATO folder.
+- [ ] If this is a new system, add a prominent [`Beta`](https://18f.gsa.gov/dashboard/stages/) label to the site
+- [ ] Ensure the staging environment has sufficient capacity to withstand the testing
+    * The testing tools create very heavy usage and traffic.
+
+##### Content
+
+- [ ] Read the LATO guide<!-- unless not doing a LATO -->.
+    * Search [this page](https://insite.gsa.gov/portal/content/627230) for "Lightweight Security Authorization Process".
+- [ ] Create a [system diagram](https://pages.18f.gov/before-you-ship/ato/ssp/#systemnetwork-diagrams)
+- [ ] Fill out the Rules of Engagement (RoE)
+    * Use staging URLs, rather than production ones.
+- [ ] Fill out the Project Plan
+- [ ] Add an [`.about.yml`](https://github.com/18F/about_yml) for the main repository
 - [ ] [Update relevant documentation](https://pages.18f.gov/before-you-ship/ato/tips/), primarily the README
-- [ ] If this will be a new ATO, add a prominent [`Beta`](https://18f.gsa.gov/dashboard/stages/) label to any currently-running sites
-- [ ] Add a [System Security Plan yml](https://pages.18f.gov/before-you-ship/ato/ssp/#template) to the repository
-- [ ] Add a System Security Plan Google doc to the ATO folder (talk to your Infrastructure Lead for a template)
+- [ ] Add a [System Security Plan YAML file](https://pages.18f.gov/before-you-ship/ato/ssp/#template) to the repository
 - [ ] [Set up Compliance Masonry documentation](https://github.com/18F/cg-compliance#starting-ato-documentation-for-cloudgov-applications)
-- [ ] [Implement the controls](https://pages.18f.gov/before-you-ship/ato/walkthrough/#step-3--implement-the-controls)
-- [ ] Add a Rules of Engagement (RoE) Google doc to the ATO folder (talk to your Infrastructure Lead for a template)
-- [ ] Ensure system environment referenced in the RoE for security testing is ready for testing. No production links should be included in the RoE so as to avoid any testing happening on production (which could lead to downtime).
-- [ ] Add a Project Plan Google doc to the ATO folder (talk to your Infrastructure Lead for a template)
 
-### Authorizing Official
+### Documentation review
+
+1. [ ] Move this issue to the `Documentation review` column of [the ATO Kanban board](https://github.com/18F/Infrastructure/projects/1). - @[infrastructure lead]
+1. [ ] Schedule a documentation review session. - @[infrastructure lead]
+    * One or more follow-up sessions may be necessary.
+1. [ ] Fix any documentation issues identified in the session.
+1. [ ] RoE signed
+    * [ ] System Owner
+    * [ ] GSA IT
+
+### Mid-Sprint
+
+1. [ ] Penetration test complete. - @[tester]
```

Comment:
Can we add a bullet for initial meeting with pen test team and/or something about it beginning? so the first bullet isn't it being complete?